Sources/OktaIdx/IDXRemediation.swift

```diff
@@ -90,6 +90,21 @@ extension IDXClient {
             super.init()
         }
         
+        public override var description: String {
+            let logger = DebugDescription(self)
+            let components = [
+                logger.address(),
+                "\(#keyPath(type)): \(type)",
```

Comment:
Can you print the type's rawValue instead? Because right now it's only showing `RemediationType` when printed.

Reply:
Done